res/figma-fonthelper-updater.service

```diff
@@ -4,7 +4,7 @@ After=systemd-user-sessions.service
 
 [Service]
 Type=simple
-ExecStart=${XDG_CONFIG_HOME}/figma-fonthelper/updater.sh
+ExecStart=/bin/bash -c '${HOME}/.local/share/figma-fonthelper/updater.sh'
```

Comment:
Can we change this to `${XDG_DATA_HOME:-$HOME/.local/share}` instead of `${HOME}/.local/share`?